grazer/src/main/java/edu/asu/diging/grazer/web/PersonController.java

```diff
@@ -64,13 +65,27 @@ public int compare(WikidataStatement o1, WikidataStatement o2) {
         return "person";
     }
     
-    @RequestMapping("/concept/{personId}/graph")
-    public String getPersonGraph(@PathVariable("personId") String personId, Model model) {
+    public boolean sameEdge(Edge edge1, Edge edge2){
+        return edge1.getSourceNode().getLabel().equals(edge2.getSourceNode().getLabel()) && 
+                edge1.getTargetNode().getLabel().equals(edge2.getTargetNode().getLabel());
```

Comment:
this should check the uri not the label. In theory the label could be the same for two different concepts.